.github/workflows/bump-version-patch.yml

```diff
@@ -0,0 +1,39 @@
+name: Create PR with bump patch version
+
+# Perform workflow based on human approval
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Bump version (patch) and create PR.
```

Comment:
Why only the patch version? I think we want flexibility of bumping the minor and major versions too

Reply:
I think major and minor versions prob require a bit more info than an automated thing. Nevertheless, those can be added later.

Reply:
But the automated thing right now is not sufficient without more info anyway – we'll introduce breaking changes in patch revisions if we can only update patch versions through this interface

Reply:
Not sure I understand what you're saying sry 🙏 . This workflow creates a PR that bumps the version (patch) instead of having to manually create it. Manual creation can still occur as normal (for major, minor, patch bumps).

Reply:
What I'm saying is if we have workflow to bump patch versions we need the same workflow for bumping minor and major versions, I don't see how the patch version component is special

Reply:
The only thing special about it is that it's more widely used, this will be the one being performed 99% of the times. I can add the other workflows but I don't feel like it should be a requirement for the PR. The bumping of versions can be still be done manually as before, this just tries to make it easier. The other versions can be added later, once we're sure this workflow is indeed working.

Reply:
It depends – if we want to follow semantic versioning here (which using the names major/minor/patch implies) then 99% of version updates will *not* be patch version updates (unless 99% of our releases are purely bugfix releases, seems unlikely).

I don't believe this should be merged with only patch version bump being automated – this will just encourage patch version updates as opposed to potentially more appropriate ones.